specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/examples/SearchServiceCreateIndex.json

```diff
@@ -339,6 +348,15 @@
                         "tempuri.org"
                     ],
                     "maxAgeInSeconds": 60
+                },
+                "encryptionKey": {
+                    "keyVaultKeyName": "myKeyName",
+                    "keyVaultKeyVersion": "myKeyVersion",
+                    "keyVaultUri": "https://myKeyVault.vault.azure.net",
+                    "accessCredentials": {
+                        "applicationId": "00000000-0000-0000-0000-000000000000",
+                        "applicationSecret": "myApplicationSecret"
```

Comment:
Per @natinimni's proposed change, `applicationSecret` should be null in the response.